napari/layers/points/_points_mouse_bindings.py

```diff
@@ -53,7 +58,8 @@ def select(layer, event):
             is_moving = True
             with layer.events.data.blocker():
                 layer._move(layer.selected_data, coordinates)
-        else:
+        elif len(event.dims_displayed) == 2:
```

Comment:
@alisterburt , I added this guard to prevent errors when clicking + dragging in select mode. I _think_ it works. Was there another guard you had in mind for click+drag?

Reply:
oh, perfect! Nope that's all we need, I was just out of date 👍🏼